stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/channel/ChannelListViewModel.kt

```diff
@@ -125,10 +125,15 @@ public class ChannelListViewModel(
     private val CHANNEL_LIMIT = 30
 
     /**
-     * Default value of the number of messages to include in each channel when querying channels,
+     * Default value of the number of messages to include in each channel when querying channels.
      */
     private val MESSAGE_LIMIT = 1
 
+    /**
+     * Default value of the number of members to include in each channel when querying channels.
+     */
+    private val MEMBER_LIMIT = 30
```

Comment:
Could we push these as const? I think I mentioned this somewhere before but it got neglected (not on this PR). 

You can put it at the bottom of the file outside the ViewModel and then make it a `const`.

Cc @bychkovdmitry if we want to put it at the bottom or top.